articles/evaluation_queues.md

```diff
@@ -0,0 +1,164 @@
+
+<img style="width: 80%;" src="/assets/images/evaluation_queue_id.png">
+
+Using this value, we can configure the `quota` parameters of this evaluation queue with the R or Python client.  
+
+{% tabs %}
+	{% tab Python %}
+		{% highlight python %}
+import synapseclient
+syn = synapseclient.login()
+evalId = 9610091
+evaluation = syn.getEvaluation(evalId)
+evaluation.quota = {'submissionLimit':3} #The maximum number of submissions per team/participant per round.
+syn.store(evaluation)
+		{% endhighlight %}
+	{% endtab %}
+
+{% tab R %}
+{% highlight r %}
+library(synapser)
+synLogin()
+
+evalId = 9610091
+evaluation <- synGetEvaluation(evalId)
+
+evaluation$quota <- c('submissionLimit'=3) #The maximum number of submissions per team/participant per round.
+synStore(evaluation)
+
+{%endhighlight %}
+{% endtab %}
+
+{% endtabs %}
+
+
+## Share an Evaluation Queue
+
+Each Evaluation has its own sharing settings.  The sharing levels are: "Administrator", "Can score", "Can Submit", and "Can view."  
+- "Administrator" sharing should be tightly restricted, as it includes authority to delete the entire Evaluation queue with all its contents. These users also have the ability to download all the submissions.
+- "Can score" allows for individuals to download all the submissions
+- "Can submit" allows for Teams or individuals to submit to the Evaluation, but doesn't have access to any of the submissions.
+- "Can view" allows for Teams or individuals to view the submissions on a leaderboard.
+
+To set the sharing setting, go to the **Challenge** tab and see your list of Evaluations.  Click on the `Share` button per Evaluation and share it with the Teams or individuals you would like.
+
+
+{% include important.html content="When someone submits to an Evaluation, a copy of the submission is made, so a person with Administrator or Can score access will be able to download the submission even if the submitter deletes the entity." %}
+
+
+## View Submissions of an Evaluation Queue
+
+All submissions of an Evaluation queue can be views through the through the use of a leaderboard.  To learn how to create a wiki page, please visit [here](http://docs.synapse.org/articles/wikis.html).  Below are instructions on how to set up a leaderboard. You must know the **evaluation Id** to do so.
+
+#### Adding Leaderboard Widget
+
+<img style="width: 80%;" src="/assets/images/add_leaderboard_widget.png">
+
+#### Configuring Leaderboard Widget
+
+Once you click on **Leaderboard**, you will have to input your own query statement such as `select * from evaluation_9610091`.  Remember, 9610091 should be replaced with your own personal evaluation Id. To view all the columns available, click **Refresh Columns**.
```

Comment:
not a personal evaluation ID, just your own.

Reply:
Fixed.